### PR TITLE
Flow-Launcher.Flow-Launcher version 1.8.0

### PR DESCRIPTION
**Pull request by jjw24**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21975)

### CODE REVIEW

File: manifests/f/Flow-Launcher/Flow-Launcher/1.8.0/Flow-Launcher.Flow-Launcher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Flow-Launcher.Flow-Launcher
+PackageVersion: 1.8.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://github.com/Flow-Launcher/Flow.Launcher/releases/download/v1.8.0/Flow-Launcher-v1.8.0.exe
+  InstallerSha256: 42856B9C0F2E0256EDD58E4E4886380E7D562E4CDBA172C706929D4BAC9E314D
```
Comment (OfficialEsco) [inline manifests/f/Flow-Launcher/Flow-Launcher/1.8.0/Flow-Launcher.Flow-Launcher.installer.yaml:10]:
```suggestion
  InstallerSha256: 42856B9C0F2E0256EDD58E4E4886380E7D562E4CDBA172C706929D4BAC9E314D
  InstallerSwitches:
    Silent: --silent
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/f/Flow-Launcher/Flow-Launcher/1.8.0/Flow-Launcher.Flow-Launcher.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Flow-Launcher.Flow-Launcher
+PackageVersion: 1.8.0
+PackageLocale: en-US
+Publisher: Flow-Launcher Team
+PackageName: Flow Launcher
+License: MIT
+LicenseUrl: https://github.com/Flow-Launcher/Flow.Launcher/blob/dev/LICENSE
```
Comment (OfficialEsco) [inline manifests/f/Flow-Launcher/Flow-Launcher/1.8.0/Flow-Launcher.Flow-Launcher.locale.en-US.yaml:10]:
```suggestion
License: MIT License
LicenseUrl: https://raw.githubusercontent.com/Flow-Launcher/Flow.Launcher/dev/LICENSE
Copyright: Copyright (c) 2019 Flow-Launcher
CopyrightUrl: https://raw.githubusercontent.com/Flow-Launcher/Flow.Launcher/dev/LICENSE
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/f/Flow-Launcher/Flow-Launcher/1.8.0/Flow-Launcher.Flow-Launcher.installer.yaml (hunk 1 above)
Comment (jjw24) [reply manifests/f/Flow-Launcher/Flow-Launcher/1.8.0/Flow-Launcher.Flow-Launcher.installer.yaml:10]: what's the change in this one?

Comment (jjw24) [reply manifests/f/Flow-Launcher/Flow-Launcher/1.8.0/Flow-Launcher.Flow-Launcher.installer.yaml:10]: it does not need the silent switch, the installation is silent by default

Comment (OfficialEsco) [reply manifests/f/Flow-Launcher/Flow-Launcher/1.8.0/Flow-Launcher.Flow-Launcher.installer.yaml:10]: It opens the Application, which is not silent.

Comment (jjw24) [reply manifests/f/Flow-Launcher/Flow-Launcher/1.8.0/Flow-Launcher.Flow-Launcher.installer.yaml:10]: It opens the application after the installation is done but has no pop up windows or require any user interaction during the installation.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**